submissions/lisnyak-o/popup-kottans/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="HandheldFriendly" content="true">
+  <meta name="MobileOptimized" content="width">
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="shortcut icon" href="favicon.png" type="image/x-icon">
+  <link
+    href="https://fonts.googleapis.com/css?family=Fira+Sans:100,100italic,200,200italic,300,300italic,regular,italic,500,500italic,600,600italic,700,700italic,800,800italic,900,900italic"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/reset.css">
+  <title>Popup HTML&CSS</title>
+</head>
+
+<body>
+  <header class="header">
+    <div class="container">
+      <nav class="menu">
+        <ul class="menu__list menu__wrapper">
+          <li class="menu__item"><a href="#" class="menu__item-link">Mail</a></li>
+          <li class="menu__item"><a href="#" class="menu__item-link">Picture</a></li>
+          <li class="menu__item"><a href="#" class="menu__item-link">
+              <label class="menu__item-btn" for="popup-checkbox">
+                <img class="menu__item-img" src="/images/popup-button.png" alt="popup-btn">
```

Comment:
![image](https://user-images.githubusercontent.com/16196199/96519483-c32e5800-1275-11eb-9040-ab0746b09b82.png)

Image links not working.

Reply:
Fixed it. Banality mistake.